Mathlib/CategoryTheory/Sites/SheafHom.lean

```diff
@@ -218,10 +218,10 @@ def sheafHom' (F G : Sheaf J A) : Cᵒᵖ ⥤ Type _ where
     exact congr_fun ((presheafHom F.1 G.1).map_comp f g) φ.1
 
 /-- The canonical isomorphism `sheafHom' F G ≅ presheafHom F.1 G.1`. -/
-def sheafHom'Iso (F G : Sheaf J A) :
+noncomputable def sheafHom'Iso (F G : Sheaf J A) :
     sheafHom' F G ≅ presheafHom F.1 G.1 :=
   NatIso.ofComponents
-    (fun _ => Equiv.toIso (equivOfFullyFaithful (sheafToPresheaf _ _))) (fun _ => rfl)
+    (fun _ => Sheaf.homEquiv.toIso) (fun _ => rfl)
```

Comment:
Why is this noncomputable now?

Reply:
It became noncomputable when `equivOfFullyFaithful` became so, but when I used `Sheaf.homEquiv` to get better defeq, I forgot to remove the tag...